### ISSUE TITLE
Poppins font is not respecting the original Indian Rupee symbol design

### ISSUE THREAD

**Issue by rakesho-vel** [google/fonts#5812]
**Describe the bug**
We are working on React Native mobile application and using the Poppins font in the app and it works really well. But recenly we are facing an issue with the Indian Rupee symbol.

The Indian Rupee Symbol is **₹** but the Poppins font is converting its bottom part to semi circular curve (pls check screenshot), which is not right as this is a specific symbol.

Can you pleas help us to make sure that the original Indian Rupee symbol looks as is with Poppins font?
 

Adding a screenshot of the Ruppee symbol in poppins font from our mobile app.

**To Reproduce**
Using Poppins font, type ₹ symbol

**Expected behavior**
The original Rupee symbol should not be covered to curved symbol as the Symbol should remain as is

**Screenshots**
Added
![Screenshot 2023-01-20 at 10 38 13 AM](https://user-images.githubusercontent.com/84490964/213684825-57b8e4fc-6892-4be5-851c-812fd3d1636c.png)

**Additional context**
Add any other context about the problem here (OS version, browser, printer…)

**Comment by xconsau** [comment]
Hello,

The rupee symbol in this font (and in many other Devanagari fonts) inherits the style of the letter "Ra". If you see the list of all available [Devanagari Fonts](https://fonts.google.com/?preview.text=%E2%82%B9&preview.text_type=custom&subset=devanagari&noto.script=Deva) and type the rupee symbol ₹ inside the preview text, you will find that the design of rupee symbol is different for each font, and inherits it's Devanagari style of "Ra". For example, check the fonts like Baloo 2, Biryani, Kadwa, Asar, etc. The rupee symbol will be quite different from it's original design. You can also choose to go with the fonts which have the rupee symbol close to the original design.

**Comment by fitojb** [comment]
I don’t think this is a bug, nor do I think that because it’s “a specific symbol”, there should be only one way to draw it. Otherwise, we would be stuck with boring perfect circles for the Euro symbol, because the European Union mandated this:

![imatge](https://user-images.githubusercontent.com/554953/213881701-22e7b67d-09ea-4e3b-9d9b-e5eb7163ba10.png)

Typefaces are, and should be, venues for artistic expression.

**Comment by rakesho-vel** [comment]
Thank you for the reply.

@xconsau , thanks for the reply, but if you look at the Devanagari carefully, Ra is "**र**" and the Rupee symbol is "**₹**". These are 2 completely different characters. And I think, we shall respect the originality of the symbol irrespective of which font type we implement.

@fitojb , yes, however boring we find the symbols to be, our font faces, should oblique to their original designs. Otherwise what is the meaning of having specifications and unique identifications to the symbol.

**Comment by applecuckoo** [comment]
Calling in @rahulgajjar for an opinion on behalf of the Indian Type Foundry (creator of Poppins)

**Comment by rahulgajjar** [comment]
Hi! There isn't any such strict guideline for the Indian rupee symbol. This is what I found regarding how the symbol should be from the reserve bank of India's website. @jonpinhorn or @satyarajpurohit would be the right people to comment on the design.

>1. What is the Indian currency called and what is its symbol?
> The Indian currency is called the Indian Rupee (INR). One Rupee consists of 100 Paise. The symbol of the Indian Rupee is ₹. The design resembles both the Devanagari letter "₹" (ra) and the Latin capital letter "R", with a double horizontal line at the top.

>[Link](https://m.rbi.org.in/scripts/FAQView.aspx?Id=136#:~:text=The%20Indian%20currency%20is%20called,horizontal%20line%20at%20the%20top.)
